tasks/users_privileges.yml

```diff
@@ -1,5 +1,10 @@
 # file: postgresql/tasks/users_privileges.yml
 
+- name: PostgreSQL | Ensure PostgreSQL is running
+  service:
+    name: "{{ postgresql_service_name }}"
+    state: started
+
 - name: PostgreSQL | Update the user privileges
```

Comment:
I think this task should be removed, because postgresql_privs can do all needed actions.